bin/prover-client/Cargo.toml

```diff
@@ -50,6 +45,8 @@ uuid.workspace = true
 strata-sp1-adapter = { workspace = true, optional = true }
 strata-sp1-guest-builder = { path = "../../provers/sp1", optional = true }
 
+strata-zkvm-hosts = { workspace = true, features = ["sp1"] }
+
```

Comment:
Why this is added here?
Shouldn't this be moved up? Or maybe add a top comment as the neighbors above and below.

Reply:
done